### DISCUSSION TITLE
Duda kata 8 f string

### DISCUSSION THREAD

**Discussion by TahiliAlv** [LaunchX-InnovaccionVirtual/CursoIntroPython#226] [Q&A] [answered]
Hola! Me atoré con este paso al imprimir los datos del diccionario, me imprime datos individuales pero en cuanto uso el f string me marca este error, no hallo que estoy haciendo mal. Alguien que me pueda orientar? Muchas gracias! 

![Screenshot_178](https://user-images.githubusercontent.com/99460744/153776302-fc1b5b04-199f-4e2c-8341-3a7dc17e5da4.png)

**Comment by OscarPedroza** [comment, marked as answer]
Hola buenas, estas usando comillas simples para la clave del diccionario y en la cadena completa también, intenta usar comillas dobles "" para que no interfiera con la cadena completa. 
Sería algo como: `planet["Name"]`

**Reply by TahiliAlv** [reply]
Woow! súper, muchas gracias! ciertamente se solucionó al reemplazar las comillas del diccionario por dobles y asegurarme que las de la cadena fueran simples n_n
